src/Overlay.tsx

```diff
@@ -71,7 +72,7 @@ const StyledOverlay = styled.div<StyledOverlayProps & SystemCommonProps & System
       opacity: 1;
     }
   }
-
+  visibility: ${props => props.visibility || 'visible'};
```

Comment:
Non-blocking: Would it be better to use a CSS variable for this (ref https://www.joshwcomeau.com/css/styled-components/#css-variables)?

Reply:
Great catch!  After a month with my head out of PRC, this didn't occur to me but is definitely a pattern I want to keep bringing in wherever possible. Fixed in 54d8783